content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+</div>
+
+
+## Setup 
+To enable Datadog Disaster Recovery, follow the relevant steps for when:
+
+1. [you are ready to configure Datadog Disaster Recovery](#when-you-are-ready-to-configure-datadog-disaster-recovery)
+2. [Datadog confirms your new organization as your secondary failover organization](#when-datadog-confirms-your-new-organization-as-your-secondary-failover-organization)
+3. [you have linked the DDR org to your primary org](#when-you-have-linked-the-ddr-org-to-your-primary-org)
+4. [you are ready to test the failover process](#when-you-are-ready-to-test-the-failover-process)
+
+#### when you are ready to configure Datadog Disaster Recovery: 
+{{% collapse-content title=" 1. Create your secondary Datadog organization" level="h5" %}}
+Identify which site your primary organization is on by matching your Datadog website URL to the [`SITE URL`][4] in the table. Then, select a secondary site for your DDR organization.
+
+{{< img src="getting_started/site/site.png" alt="The site URL in your browser tab" style="width:40%" >}}
+
+| Site    | Site URL                    | Site Parameter      | Location |
+|---------|-----------------------------|---------------------|----------|
+| US1     | `https://app.datadoghq.com` | `datadoghq.com`     | US       |
+| US3     | `https://us3.datadoghq.com` | `us3.datadoghq.com` | US       |
+| US5     | `https://us5.datadoghq.com` | `us5.datadoghq.com` | US       |
+| EU1     | `https://app.datadoghq.eu`  | `datadoghq.eu`      | EU (Germany) |
+| AP1     | `https://ap1.datadoghq.com` | `ap1.datadoghq.com` | Japan |
+
+For example, if you are hosted in [US1](https://app.datadoghq.com), you may choose to select another Datadog site to ensure observability continuity in the event of a regional disaster. All Datadog sites are geographically separated.
+
+Contact your [Customer Success Manager](mailto:success@datadoghq.com) or [Datadog Support](https://www.datadoghq.com/support/) to help you select the secondary Datadog site and configure your new organization to be your secondary failover organization.
+
+If you're also sending telemetry to Datadog using cloud provider integrations, you must add your cloud provider accounts in the secondary org. 
+
+Datadog does not use cloud providers to receive telemetry data while the secondary site is passive.
+
+**Note**: Datadog can set this up for you if you'd prefer. 
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 2. Contact Datadog to share your new organization" level="h5" %}}
+Share your organization name with your Datadog [Customer Success Manager](mailto:success@datadoghq.com) so they can configure your new organization to be your secondary failover organization.<br><br>
+**Note**: This organization appears in your Datadog billing hierarchy, but all usage and cost associated is _not_  billed during the private beta.
```

Comment:
The top of the page says it's in Preview, but this says it's in private beta - are we using both terms for a reason?